wordpress-mobile/WordPress-iOS
# Stop using simple previews as fallback

When a preview fails to load, we used to fall back to a "fake" preview with an message on top. Instead of that, this just shows that an error happened and a button to retry.

Besides that, it will detect when the connection comes back and retry loading automatically

![9285-beforeafter](https://user-images.githubusercontent.com/8739/41111654-da6d0bc2-6a7c-11e8-964e-fba8454d6251.png)

Fixes #9285 

To test:

1. Find a post that can be previewed (no local changes).
2. Put your phone on airplane mode, or turn off the wifi if you're using a simulator.
3. Try to view/preview the post, it should show an error message.
4. Bring your phone/simulator back online.
5. The preview should load automatically.
